mirage/serializers/user.ts

```diff
@@ -32,12 +32,24 @@ export default class UserSerializer extends ApplicationSerializer<User> {
                     },
                 },
             },
+            default_region: {
+                data: {
+                    type: 'regions',
+                    id: 'us',
+                },
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/regions/us/`,
+                        meta: {},
+                    },
+                },
+            },
         };
     }
 
     buildNormalLinks(model: ModelInstance<User>) {
         return {
-            ...super.buildNormalLinks(model),
+            self: `${apiUrl}/v2/users/${model.id}/`,
```

Comment:
Is there a reason `super.buildNormalLinks` won't work here?

Reply:
That's a thing I didn't catch was a recent change when I broke out this code. We'll go back to buildNormalLinks.